store/store_download.go

```diff
@@ -475,6 +476,10 @@ func downloadImpl(ctx context.Context, name, sha3_384, downloadURL string, user
 			pbar = progress.Null
 		}
 		dlSize = float64(resp.ContentLength)
+		if dlSize == 0 {
+			logger.Noticef("unexpected Content-Length: 0 for %s", downloadURL)
+		}
+		logger.Noticef("Download size for %s: %d", downloadURL, resp.ContentLength)
```

Comment:
shouldn't that be "download" as you made the other logs lowercase?

Reply:
The rest of the changes are about error messages (which should be lowercase per go convention). But yes I made a mistake with lowercase "unexpected" above.